extensions/widgets/segment_new/segment_new.lcb

```diff
@@ -0,0 +1,716 @@
+/*
+Copyright (C) 2015 Runtime Revolution Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+/*
+Description: This widget is a segmented control.
+A segmented control is a horizontal control that is made up of multiple segments, where each segment functions as a discrete button.
+*/
+
+-- declaring extension as widget, followed by identifier
+widget com.livecode.extensions.livecode.segmented
+--
+
+-- adding metadata to ensure the extension displays correctly in livecode
+metadata title is "Segmented"
+metadata author is "Georgia Hutchings"
+metadata version is "1.0.0"
+--
+
+-- dependency declarations
+use com.livecode.canvas
+use com.livecode.widget
+use com.livecode.engine
+--
+
+-- property declarations
+property theme				get mTheme				set setTheme
+property segmentStyle 		get mStyle				set setStyle			-- string (enum) : segment or bar
+property multiSelect		get mMultiSelect		set setMultiSelect		
+
+--property segementNames		get getSegmentNames		set setSegmentNames		-- list of optional strings
+property segmentNames		get mSegmentNames		set setSegmentNames
+
+property segmentLabels		get getSegmentLabels	set setSegmentLabels	-- list of optional strings
+property segmentIcons		get getSegmentIcons		set setSegmentIcons		-- list of optional strings
+property selectedIcons		get getSelectedIcons	set setSelectedIcons	-- list of optional strings
+
+property segmentDisplay		get mSegmentDisplay		set setSegmentDisplay	-- string (enum) : text or icon
+property segmentMinWidth	get getSegmentMinWidth	set setSegmentMinWidth	-- list of optional number
+property selectedSegment	get getSelectedSegment	set setSelectedSegment	
+--
+
+-- private instance variables
+--properties
+private variable mTheme 			as string
```

Comment:
Isn't used in the rest of the widget and will be replaced when we use Fraser's library for theming. We don't want to confuse users with theme properties when DP1 goes live.